engine/worker/cmd_register.go

```diff
@@ -30,7 +30,7 @@ func cmdRegisterRun(w *currentWorker) func(cmd *cobra.Command, args []string) {
 			ModelID:      w.model.ID,
 		}
 
-		if err := w.register(form); err != nil {
+		if err := w.register(form, viper.GetBool("auto_update")); err != nil {
```

Comment:
why do we want to run `worker register --auto-update` ?
i suppose we can do register with auto-update to false here